src/main/java/com/rednavis/vaadin/showcase/views/ConstantUtils.java

```diff
@@ -6,4 +6,10 @@
 
   public static final String PAGE_LOGIN_TITLE = "Login";
   public static final String PAGE_LOGIN_URL = "loginView";
+  public static final String LOGIN_FORM_LOGIN = "Username";
```

Comment:
This class is for application internal use only.
Localization must be moved to a separate class

Reply:
Yes, I know, it was added constants without settings of localisation now. And it is another task.